release/nightly_tests/nightly_tests.yaml

```diff
@@ -221,7 +221,7 @@
 
   run:
     timeout: 7200
-    script: python stress_tests/test_dead_actors.py
+    script: python stress_tests/test_dead_actors.py --num-threads=5
```

Comment:
Why don't we actually keep the original test (as it is the most common use case) and have a separate tests for threaded actors?